concepts/05 UI Components/DropDownBox/03 Overview/10 Key Features.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
Let's organize the list of features the following way
1. Synchronize with Embedded Component
2. Single and Multiple Selection 
3. Validation (isDirty, isValid, all validation properties)
4. Custom components ([contentTemplate](https://js.devexpress.com/React/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#contentTemplate), [fieldTemplate](https://js.devexpress.com/React/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#fieldTemplate)
5. Input Field Customization (please check we use Text Field and Input field in different places, this is pretty the same but better to use one term anyway)
6. A11y support & RTL & KBN

I agree that we can get rid of the 'Extended Functionality' section here